src/client/helpers/setupIconLibrary.js

```diff
@@ -38,6 +38,9 @@ config.autoAddCss = true;
 // Add Icons to FontAwesome library
 library.add(
 	fab, faAngleDoubleLeft, faAngleDoubleUp, faBook, faCalendarAlt, faChartLine, faCheck, faCircle,
+	faCircleNotch, faCodeBranch, faComment, faCommentDots, faComments, faEnvelope, faExclamationTriangle,
+	faExternalLinkAlt, faGlobe, faGripVertical, faHistory, faInfo, faListUl, faPenNib, faPencilAlt, faPencilRuler,
+	faPlus, faQuestionCircle, faRemoveFormat, faSave, faSearch, faSignInAlt,
```

Comment:
It looks like these lines were duplicated during the merge from master.
You'll need to make sure the result does have the 3 new icons in there (faComment, faGripVertical and faSave)